test/fake/plugin_server.go

```diff
@@ -1,4 +1,4 @@
-package e2e
+package fake
```

Comment:
I cannot put the `fake` pkg inside `e2e` as then logs streaming for test doesn't work because of this issue: https://github.com/golang/go/issues/46959